src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
@barbieri To support that setup, I think it would be nice for swupd to understand a prioritized config scheme, /usr/share/defaults/swupd/??-versionurl, where swupd will choose the config the greatest priority. That way, you can have different bundles for every quality level, each of which installs a different ??-versionurl file. Would this scheme work for you?